### ISSUE TITLE
No progress indicator for image uploads from a comment

### ISSUE THREAD

**Issue by miketaylr** [webcompat/webcompat.com#1847]
(STR taken from https://github.com/webcompat/webcompat.com/issues/1000#issuecomment-339006338)

1. Go to https://staging.webcompat.com/issues/1294
2. Add a comment with an image around 3Mb (desktop screenshot for example)
3. Start your chrono ;)

Basically because it takes a long time and there is no visual cue to tell the user that an upload is in the process, you might give up or continue typing text or even restart an upload. 

Sometimes it has happen to me that I posted the text comment without remembering that I started an upload. And the markup for the image shows up in the next comment form. :)

**Comment by miketaylr** [comment]
We could do something quick and dirty to be less confusing, then design a nicer solution. Something like:

![screen shot 2017-10-24 at 10 14 10 am](https://user-images.githubusercontent.com/67283/31951769-95c20e3a-b8a4-11e7-840b-06780f99eaee.png)

(re-using the existing progress indicator for new issue). WDYT @karlcow @zoepage? The code can probably be re-used from the form.

**Comment by miketaylr** [comment]
(imagine that yellow circle is the progress gif thingy: https://webcompat.com/img/upload-loader.svg)

<img src="https://webcompat.com/img/upload-loader.svg">

**Comment by karlcow** [comment]
yup that's it.

**Comment by zoepage** [comment]
That makes sense :) Cool! There is already a [SVG icon](https://github.com/webcompat/design/blob/master/docs/svg/icons/spinner8.svg) for that in the design repo we'll use.

**Comment by miketaylr** [comment]
Is there an animated version of that @zoepage?

**Comment by zoepage** [comment]
@miketaylr no, but with CSS it's pretty easy to animate. If you feel like this is not what you want, we can create an animated version.

**Comment by zoepage** [comment]
Today, I had an idea for a pretty different approach to a loader.

How about we do not use a spinner, but animate the affected area* like this:
https://codepen.io/zoepage/pen/mpapVK

* e.g. 
upload -> upload area
comments -> comment area
...

(adding @magsout as he is not in this discussion yet)

**Comment by magsout** [comment]
@zoepage I like the idea with what I call placeholder. 👍 

So, yes I like your idea ;)

**Comment by karlcow** [comment]
My 💴 💴.
I'm not sure people would understand something is happening. Better than what we have currently at least. So mild +1
A spin https://codepen.io/anon/pen/rpgmOa
Also https://dribbble.com/search?q=upload+progress

**Comment by denschub** [comment]
I do agree with Karl, if the upload area flashes, it probably won't be enough. But if we flash and change the text content to "Uploading..." or something, that should be very nice. :)

**Comment by zoepage** [comment]
I love the "we are compacting..." :D

**Comment by zoepage** [comment]
This will be tackled after #2011 and #2012 are done.

**Comment by zoepage** [comment]
merged #2259
